doc/SpectatorView.Setup.DSLR.md

```diff
@@ -1 +1,164 @@
+    2. Open the SpectatorView.Example.Unity project.
+    3. Open the SpectatorView.ExtrinsicsCalibration Unity scene.
+    4. Open the Calibration window in the Unity Editor. This can be found in the toolbar under 'Spectator View' -> 'Calibration'.
+    ![Marker](images/CalibrationToolbar.png)
+    5. Run the SpectatorView.ExtrinsicsCalibration Unity scene in the editor.
+    6. Connect to your HoloLens 2 device by specifying its IP address in the Holographic Camera text field and pressing 'Connect'.
+    7. Press 'Request Marker Data' to kick off QR Code detection.
+    8. View your printed [Calibration Board](images/CalibrationBoard.png) through the HoloLens 2 device and make sure that blue and green squares appear over the QR Codes and ArUco markers (This may require getting the HoloLens 2 extremely close to the calibration board. 5cm QR Codes need to be detected from a distance of 10cm).
+
+>Note: Don't skip this testing step. You will need to accept the webcam permission dialogue that is launched when first attempting QR Code detection with the newly installed SpectatorView.HolographicCamera application. Once the HoloLens 2 is attached to the DSLR rig, it will be extremely difficult/impossible to accept these dialogues without dismantling your rig.
+
+12. Attach your DSLR camera to the PC capture card and ensure that the camera stream works.
+    1. Attach your DSLR camera by HDMI or SDI to the capture card hooked up to your PC.
+    2. Turn on your DSLR camera.
+    3. Open the SpectatorView.Example.Unity project.
+    4. Open the SpectatorViewCompositor Unity scene in the Unity Editor.
+    5. Open the Compositor window in the Unity Editor. This can be found in the toolbar under 'Spectator View' -> 'Compositor'.
+    ![Marker](images/CompositorToolbar.png)
+    6. Run the SpectatorViewCompositor scene in the Unity Editor.
+    7. You should see the Camera Feed appear in the Compositor window if everything has been configured correctly.
+
+13. Attach your HoloLens 2 device to the DSLR Camera using the HoloLens 2 camera mount (Coming soon...)
+
+## Camera Intrinsics
+
+Camera intrinsics quantify focal lengths, principal points and lens distortion information for your DSLR camera. For DSLR filming, camera intrinsics are used to calculate the Unity camera's projection matrix for the hologram feed. This then allows the hologram feed to be composited with the DSLR camera feed. For more information on camera intrinsics, see [here](https://en.wikipedia.org/wiki/Camera_resectioning).
+
+>Note: Changing the zoom and focus length of your DSLR camera's lens will change the camera intrinsics. When conducting calibration, you should change your lens to its manual focus setting. You should also avoid using the manual focus ring to change focus when obtaining images. Although images may at times appear blurry, as long as the are kept at a consistent focus length, you should be able to obtain a usable camera intrinsics.

```

Comment:
>Although images may at times appear blurry, as long as the are kept at a consistent focus length, you should be able to obtain a usable camera intrinsics. [](start = 279, length = 154)

I think it might be worth adding something like this: "For best results, use a bright external light to illuminate your calibration board. Set your lens's aperture to the smallest aperture (largest f-stop value) for which images still are exposed enough to see the marker pattern. Smaller apertures increase the depth of field, and will allow for less focus blur when capturing images at different depths."